src/ApiClient.php

```diff
@@ -147,12 +147,12 @@ public function getRequestStatus(string $permanentId): TranslationResponse
     /**
      * @inheritDoc
      */
-    public function getTranslatedFiles(string $permanentId): array
+    public function getFile(string $fileUrl): string
```

Comment:
This can be used not only to download translated files. There are other types of assets, like source and referenced files.